superset/viz.py

```diff
@@ -840,8 +841,20 @@ def get_data(self, df: pd.DataFrame) -> VizData:
         for metric in metrics:
             aggfuncs[metric] = self.get_aggfunc(metric, df, self.form_data)
 
-        groupby = self.form_data.get("groupby")
-        columns = self.form_data.get("columns")
+        groupby = self.form_data.get("groupby") or []
+        columns = self.form_data.get("columns") or []
+
+        def _format_datetime(value: Any) -> Optional[str]:
+            if isinstance(value, str):
+                return f"__timestamp:{datetime_to_epoch(pd.Timestamp(value))}"
```

Comment:
@villebro it seems this only supports unix timestamp data type, not `datetime`. E.g. column generated from `SELECT now()` or `SELECT date_trunc(TIMESTAMP '2020-01-01 00:00:00')` in PostgreSQL.